app/src/main/AndroidManifest.xml

```diff
@@ -30,7 +30,7 @@
         -->
         <meta-data
             android:name="com.google.android.geo.API_KEY"
-            android:value="${YOUR_API_KEY}" />
+            android:value="" />
```

Comment:
This same suggestion appeared in a previously submitted code review.
When that occurs, only the first gets credit.
See my comments there.  Summarized: having this here is a good reminder of where to put the API key.  It's best to find a way to hide the API key from GitHub.